docs/modules/globalfee.md

```diff
@@ -1,26 +1,28 @@
 # Gaia Fees and Fees Checks
 
 ## Fee Parameters
-The CosmosHub allows managing fees using 3 parameters:
+The CosmosHub allows managing fees requirement using 4 parameters:
```

Comment:
I think we should rework this section, it's a bit confusing

```
`MinimumGasPricesParam`
`BypassMinFeeMsgTypes` 
`MaxTotalBypassMinFeeMsgGasUsage`
```

These are gov parameters - they MUST BE same all network nodes

```
minimum-gas-prices
```
Is a config param that allows operators to additionally configure their nodes.